int8 thrust_factor
int8 yaw_setpoint
int8 depth_setpoint
int8 speed_change
